firefox_accounts/views/fxa_flow_aggregates.view.lkml

```diff
@@ -3,10 +3,7 @@
 
 view: fxa_flow_aggregates {
   derived_table: {
-    interval_trigger: "24 hours"
-    increment_key: "flow_start_ts"
```

Comment:
I think the problem might be that the SQL query does not have an `incrementcondition`: https://cloud.google.com/looker/docs/reference/param-view-increment-key#creating_an_incremental_sql-based_pdt I basically return the same set of data (plus some data that got added) during every interval.